javascript_nodejs/81.facebook-events/index.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { BotFrameworkAdapter, BotStateSet, ConversationState, MemoryStorage, UserState } = require('botbuilder');
+const { BotConfiguration } = require('botframework-config');
+const { DialogSet } = require('botbuilder-dialogs');
+const path = require('path');
+const restify = require('restify');
+const FacebookEventsBot = require('./bot');
+
+const CONFIG_ERROR = 1;
+
+// Read botFilePath and botFileSecret from .env file.
+// Note: Ensure you have a .env file and include botFilePath and botFileSecret.
+const ENV_FILE = path.join(__dirname, '.env');
```

Comment:
Our other JS samples support reading appId / password from environment variables. Consider following those patterns here. (Unsless I'm missing something obvious)

Reply:
It is supported:

// Create adapter. See https://aka.ms/about-bot-adapter to learn more about adapters.
const adapter = new BotFrameworkAdapter({
    appId: endpointConfig.appId || process.env.MicrosoftAppId,
    appPassword: endpointConfig.appPassword || process.env.MicrosoftAppPassword
});